scripts/modules/b2b-account/quotes.js

```diff
@@ -335,16 +346,17 @@ define([
                 index: 'status',
                 displayName: 'Status',
                 sortable: false
+               
             }
         ],
         rowActions: [
             {
-                displayName: 'Edit Quote',
-                action: 'editQuote'
+                displayName: 'View Quote',
```

Comment:
We also need to take this, displayName from label files. If you wanted to get values from labels files in javascript then you can use the following. Not sure this will work on not but give it a try. take the localized labels.

Hypr.getLabel('key')

Reply:
Yes Its working. Done